ui/apps/platform/src/Containers/Policies/PatternFly/Table/PoliciesTable.tsx

```diff
@@ -118,6 +126,8 @@ function PoliciesTable({
     const [deletingIds, setDeletingIds] = useState<string[]>([]);
     const [isDeleting, setIsDeleting] = useState(false);
 
+    const [enableDisableType, setEnableDisableType] = useState<EnableDisableType>('');
```

Comment:
Using empty string to represent "false" is a little confusing to me. A type union with `null` may be clearer.

```suggestion
    const [enableDisableType, setEnableDisableType] = useState<EnableDisableType | null>(null);
```